tests/tools/create_chunked_dataset.py

```diff
@@ -0,0 +1,275 @@
+
+        paper_val_mask_path = os.path.join(input_dir, 'paper/val_mask.npy')
+        with open(paper_val_mask_path, 'wb') as f:
+            np.save(f, paper_val_mask)
+
+        author_train_mask_path = os.path.join(input_dir, 'author/train_mask.npy')
+        with open(author_train_mask_path, 'wb') as f:
+            np.save(f, author_train_mask)
+
+        author_test_mask_path = os.path.join(input_dir, 'author/test_mask.npy')
+        with open(author_test_mask_path, 'wb') as f:
+            np.save(f, author_test_mask)
+
+        author_val_mask_path = os.path.join(input_dir, 'author/val_mask.npy')
+        with open(author_val_mask_path, 'wb') as f:
+            np.save(f, author_val_mask)
+
+        inst_train_mask_path = os.path.join(input_dir, 'institution/train_mask.npy')
+        with open(inst_train_mask_path, 'wb') as f:
+            np.save(f, inst_train_mask)
+
+        inst_test_mask_path = os.path.join(input_dir, 'institution/test_mask.npy')
+        with open(inst_test_mask_path, 'wb') as f:
+            np.save(f, inst_test_mask)
+
+        inst_val_mask_path = os.path.join(input_dir, 'institution/val_mask.npy')
+        with open(inst_val_mask_path, 'wb') as f:
+            np.save(f, inst_val_mask)
+
+        node_data = {
+                    'paper':
+                    {
+                        'feat': paper_feat_path,
+                        'train_mask': paper_train_mask_path,
+                        'test_mask': paper_test_mask_path,
+                        'val_mask': paper_val_mask_path,
+                        'label': paper_label_path,
+                        'year': paper_year_path
+                    },
+                    'author':
+                    {
+                        'train_mask': author_train_mask_path,
+                        'test_mask': author_test_mask_path,
+                        'val_mask': author_val_mask_path
+
+                    },
+                    'institution':
+                    {
+                        'train_mask': inst_train_mask_path,
+                        'test_mask': inst_test_mask_path,
+                        'val_mask': inst_val_mask_path
+                    }
+                }
+    else:
+        node_data = {
+                    'paper': {
+                        'feat': paper_feat_path,
+                        'label': paper_label_path,
+                        'year': paper_year_path,
+                    }
+                }
+
+    output_dir = os.path.join(root_dir, 'chunked-data')
+    chunk_graph(
+            g,
+            'mag240m',
+            node_data,
+            {
+                'cites': {'count': cite_count_path},
+                'writes': {'year': write_year_path},
+                'rev_writes': {'year': write_year_path},
+            },
+            num_chunks=num_chunks,
+            output_path=output_dir,
+    )
+    print('Done with creating chunked graph')
+
+    # check metadata.json
```

Comment:
the check for output of `chunk_graph` should be added into test script, as a testcase. we don't need to always check it in each call of `create_chunked_dataset()`

Reply:
Changed code as per our slack channel chat.